doozer/doozerlib/cli/olm_bundle.py

```diff
@@ -140,18 +145,37 @@ def rebase_and_build_olm_bundle(runtime: Runtime, operator_nvrs: Tuple[str, ...]
         cluster-logging-operator-container-v4.2.30-202004240858
     """
 
-    runtime.initialize(config_only=True)
-    clone_distgits = bool(runtime.group_config.canonical_builders_from_upstream)
-    runtime.initialize(clone_distgits=clone_distgits)
-
-    if not operator_nvrs:
-        # If this verb is run without operator NVRs, query Brew for all operator builds
+    if runtime.images and operator_nvrs:
+        raise click.BadParameter("Do not specify operator NVRs when --images is specified")
+
+    operator_builds = []
+    if operator_nvrs:
+        LOGGER.info("Fetching given nvrs from brew...")
+        brew_session = koji.ClientSession(BREW_HUB)
+        images = []
+        for index, build in enumerate(brew.get_build_objects(operator_nvrs, brew_session)):
+            if not build:
+                raise IOError(f"Build {operator_nvrs[index]} doesn't exist in Brew.")
+            source_url = urlparse(build['source'])
+            image_name = source_url.path.rstrip('/').rsplit('/')[-1]
```

Comment:
This is a little bit concerning... But it is fine unless the assumption breaks.

Reply:
I agree, it is not perfect. I copied that part from how we are determining it in OLMBundle class, so I think it has been working without breaking for all operator images. I will look if I can replace it with something more robust in another PR.